GillianCore/GIL_Parser/GIL_Lexer.mll

```diff
@@ -262,8 +262,11 @@ rule read = parse
   | loc                  { GIL_Parser.LOC (Lexing.lexeme lexbuf) }
   | aloc                 { GIL_Parser.ALOC (Lexing.lexeme lexbuf) }
   | normalised_aloc      { GIL_Parser.ALOC (Lexing.lexeme lexbuf) }
-
-  (* Variables: THIS IS NEW *)
+(* Directives *)
+  | "@nopath"            { GIL_Parser.NO_PATH }
+  | "@internal"          { GIL_Parser.INTERNAL }
+  | "internal;"          { GIL_Parser.INTERNAL_FILE }
```

Comment:
Minor, but can we go with
`@@internal` or `#internal` and drop the `;` at the end ?
I slightly prefer the second one because the syntax already exists in the wild (C)